rpc/lib/server/handlers.go

```diff
@@ -538,7 +538,7 @@ func (wsc *wsConnection) writeRoutine() {
 				return
 			}
 		case msg := <-wsc.writeChan:
-			jsonBytes, err := json.Marshal(msg)
+			jsonBytes, err := json.MarshalIndent(msg, "", "  ")
```

Comment:
Can't do this with websockets.

I find it very nice to work with api's like this.  Especially when you check them with the webpage.  Just more readable.  And it is not much more overhead.  If you use gzip compression (if you care about bandwidth, we can add a middleware), then it goes down to a few % overhead).

Maybe this is just my aethetics going overboard.  But I see no reason not to make it more readable.